src/functions-and-arrays.js

```diff
@@ -28,9 +109,38 @@ const wordsUnique = [
   'simple',
   'bring'
 ];
+const uniquifyArray = wordsUnique => {
+  if (!wordsUnique.length) { return null; }
+  let cleanedArray = [];
+  for (let i = 0; i < wordsUnique.length; i++) {
+    if (isWordPresent(cleanedArray, wordsUnique[i])) {
+      continue;
+    } else {
+      cleanedArray.push(wordsUnique[i]);
+    }
+  }
+  return cleanedArray;
+}
+
+const isWordPresent = (cleanedArray, word) => {
+  if (cleanedArray.indexOf(word) === -1) { return false }
+  else { return true; }
+}
+
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
+const doesWordExist = (wordsFind, word) => {
+  if (!wordsFind.length) {return null;}
+  let found = false;
+  for (let i = 0; i < wordsFind.length; i++) {
+    if (wordsFind[i] === word) {
+      found = true;
+    }
+  }
+  if (found === true) { return true; } else { return false; }
```

Comment:
Good work Elia! I think if you have on line 141 `return found` it would be simpler